src/components/PatientsTable/PatientMenu.tsx

```diff
@@ -0,0 +1,57 @@
+//
+// This source file is part of the Stanford Biodesign Digital Health Spezi Web Health Components open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import { type Patient } from '@medplum/fhirtypes'
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+} from '@stanfordspezi/spezi-web-design-system/components/DropdownMenu'
+import { ConfirmDeleteDialog } from '@stanfordspezi/spezi-web-design-system/molecules/ConfirmDeleteDialog'
+import { useOpenState } from '@stanfordspezi/spezi-web-design-system/utils/useOpenState'
+import { Link } from '@tanstack/react-router'
+import { Pencil, Trash } from 'lucide-react'
+
+interface PatientMenuProps {
+  onDelete: (data: Patient) => void
+  editRoute: string
+  data: Patient
+}
```

Comment:
If it's already in "patient" context, it seems to make sense to adjust names to reflect what data is passed.

```suggestion
interface PatientMenuProps {
  onDelete: (patient: Patient) => void
  editRoute: string
  patient: Patient
}
```